app/views/admin/api_docs/base/_form.html.slim

```diff
@@ -15,7 +15,7 @@
   = form.semantic_errors :base_path
   = form.input :skip_swagger_validations, as: :boolean
 = form.buttons do
-  = form.commit_button
+  = form.commit_button 'Create Spec'
```

Comment:
If not from here, where is coming from the "Create Service" text in the button?

Reply:
It's inferred from the model, the form is for `ApiDocs::Service`: https://github.com/3scale/porta/blob/9f413e8361f2752c47c44b4b9beaab27f5c0669a/app/views/admin/api_docs/base/new.html.erb#L3